core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -261,16 +259,25 @@ private void loginAndTakeBack(StaplerRequest req, StaplerResponse rsp, User u) t
     }
 
     /**
-     * Creates an user account. Used by admins.
+     * Creates a user account. Used by admins.
      *
      * This version behaves differently from {@link #doCreateAccount(StaplerRequest, StaplerResponse)} in that
      * this is someone creating another user.
      */
     public void doCreateAccountByAdmin(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
+        createAccountByAdmin(req, rsp, "addUser.jelly", "."); // send the user back to the listing page on success
+    }
+
+    /**
+     * Creates a user account. Requires {@link Jenkins#ADMINISTER}
+     */
+    public User createAccountByAdmin(StaplerRequest req, StaplerResponse rsp, String addUserView, String successView) throws IOException, ServletException {
```

Comment:
> Methods named doSomething are implicit web methods, i.e. really exist only for Stapler. That's how we justify changing/deleting them

@daniel-beck don't like that implicit difference, really, but fair... I suppose it stems from the mix of UI with business logic, anyway hope this is all set now